beacon_node/network/src/beacon_processor/worker/gossip_methods.rs

```diff
@@ -787,19 +785,15 @@ impl<T: BeaconChainTypes> Worker<T> {
         // propagated on the gossip network.
         self.propagate_validation_result(message_id, peer_id, MessageAcceptance::Accept);
 
-        /* TODO
-        // Register the attestation with any monitored validators.
         self.chain
             .validator_monitor
             .read()
-            .register_gossip_aggregated_attestation(
+            .register_gossip_sync_committee_contribution(
                 seen_timestamp,
-                aggregate.aggregate(),
-                aggregate.indexed_attestation(),
+                sync_contribution.aggregate(),
+                sync_contribution.participant_pubkeys(),
                 &self.chain.slot_clock,
             );
```

Comment:
should we add the `BEACON_PROCESSOR_SYNC_CONTRIBUTION_VERIFIED_TOTAL` metric here? I don't see it used elsewhere

Reply:
Thanks.